### PR TITLE
fix(deploy): use semantic release for deployment to PyPI

### PR DESCRIPTION
**Pull request by chriswmackey**

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
@chriswmackey this looks good but in order to get it all fixed similar to other repositories you need to:

1. update `.travis.yml` file to use the library from npm and not the python one. this will ensure that we always use the latest and the greatest version.

2. add the `.releaserc.json` file to set-up semantic-release plugins. Here is an example: https://github.com/ladybug-tools/honeybee-radiance/blob/master/.releaserc.json

3. the previous file uses `deploy.sh` to deploy the libraries to PyPI and also generate the documentation. You need to also add the `deploy.sh` file. The only change you need to make is the folder for documentation. If you want the document not to be generated then just comment it out. https://github.com/ladybug-tools/honeybee-radiance/blob/master/deploy.sh

4. You also need to update requirements.txt and add dependencies needed for deployment and documentation:  https://github.com/ladybug-tools/honeybee-radiance/blob/3541b5da339e5319dd9d90bf3fc0824c0f84ee34/dev-requirements.txt#L6-L10

**Comment by chriswmackey** [comment]
Tahnks for all of the instructions @mostaphaRoudsari . I think that I followed them correctly and that will be great to have the documentation auto-generated for the library. Please let me know if it all looks ok.

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+if [ -n "$1" ]
+then
+  NEXT_RELEASE_VERSION=$1
+else
+  echo "A release version must be supplied"
+  exit 1
+fi
+
+# deploy_to_pypi
```
Comment (mostaphaRoudsari) [inline deploy.sh]: You should uncomment this line if you want the deployment to PyPI to happen.

```suggestion
deploy_to_pypi
```

**Comment by chriswmackey** [comment]
Ok. All has been fixed with regards to docs and the comment in the .sh file.

**Pull request merged**